infra/license/license-checker.ts

```diff
@@ -31,6 +31,134 @@ var args = process.argv.slice(2)
 const inputPathOfLicenseList = args[0];
 const verificationTag = args[1];
 
+namespace Verification {
+
+const resultTypeDescription = {
+  pass: 'No license issue found',
+
+  warnConditionalLicense: 'Additional manual verification is needed',
+  warnNeverChecked: 'Never checked license is detected',
+  warnUnknownLicense: 'License is not found',
+
+  failForbiddenLicense: 'Forbidden license is found',
+  failForbiddenPackage: 'Forbidden package is used'
+} as const ;
```

Comment:
Formatter passed this one?
```suggestion
} as const;
```

Reply:
I was surprised too because... formatter changed as it is... @.@